CheckForVitals.cpp

```diff
@@ -0,0 +1,30 @@
+#include "./CheckForVitals.h"
+#include "./PrintMessage.h"
+
+bool CheckForRange(float value, float minimum, float maximum) {

```

Comment:
It's better to name boolean-return functions with "is". For example, `isInRange`
Otherwise, `checkForRange` can mean "return true if within range", or "return true when attention is needed (out of range"